google/cloud/spanner/samples/samples.cc

```diff
@@ -3649,6 +3666,250 @@ void DropForeignKeyConstraintDeleteCascade(
 }
 // [END spanner_drop_foreign_key_constraint_delete_cascade]
 
+// [START spanner_add_proto_type_columns]
+void AddProtoTypeColumns(
+    google::cloud::spanner_admin::DatabaseAdminClient client,
+    std::string const& project_id, std::string const& instance_id,
+    std::string const& database_id) {
+  google::cloud::spanner::Database database(project_id, instance_id,
+                                            database_id);
+  google::spanner::admin::database::v1::UpdateDatabaseDdlRequest request;
+  request.set_database(database.FullName());
+  google::protobuf::FileDescriptorSet fds;
+  google::cloud::spanner::testing::SingerInfo::default_instance()
+      .GetMetadata()
+      .descriptor->file()
```

Comment:
I assume we can assume this is never `nullptr`, or that if it is, something else has gone wrong.

Reply:
My recollection is that it can indeed never be null.